### PR TITLE
Build script to build and test package restic.

### PR DESCRIPTION
**Pull request by vikasgupta8**
Build script to build and test package restic.

### CODE REVIEW

File: r/restic/restic_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : restic
+# Version       : v0.12.1
+# Source repo   : https://github.com/restic/restic/
+# Tested on     : rhel UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Santosh Magdum <santosh.magdum@us.ibm.com>
```
Comment (seth-priya) [inline r/restic/restic_ubi8.4.sh:8]: Please use your name and email ID in the maintainer's field

File: r/restic/restic_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : restic
+# Version       : v0.12.1
+# Source repo   : https://github.com/restic/restic/
+# Tested on     : rhel UBI 8.4
```
Comment (seth-priya) [inline r/restic/restic_ubi8.4.sh:6]: If it has been tested on both RHEL and UBI, please specify both distros with versions as a comma separated list, example: "RHEL 8.4, UBI 8.4"

File: r/restic/restic_ubi8.4.sh [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# Script License: Apache License, Version 2 or later
+# Maintainer's  : Santosh Magdum <santosh.magdum@us.ibm.com>
+#                 Priya Seth <priya.seth@us.ibm.com>
+#
```
Comment (seth-priya) [inline r/restic/restic_ubi8.4.sh:10]: Please refer to the updated templates in the "templates" folder as there are additional fields like Language that now need to be supplied.

File: r/restic/restic_ubi8.4.sh [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_NAME=restic
+PACKAGE_PATH=github.com/restic/restic
```
Comment (seth-priya) [inline r/restic/restic_ubi8.4.sh:21]: looks like we are diverging from the template and the variables it sets here https://github.com/ppc64le/build-scripts/blob/66d9be10cc51eecd1859e81cc3f27d17cd4fca25/templates/build_script_go.sh#L20 , any specific reason for that?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: r/restic/restic_ubi8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply r/restic/restic_ubi8.4.sh]: Updated to UBI 8.4.

File: r/restic/restic_ubi8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply r/restic/restic_ubi8.4.sh]: Done.

File: r/restic/restic_ubi8.4.sh [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
```
[the remark this answers is not in the document]
Comment (vikasgupta8) [reply r/restic/restic_ubi8.4.sh:19]: Added.

File: r/restic/restic_ubi8.4.sh (hunk 3 above)
Comment (vikasgupta8) [reply r/restic/restic_ubi8.4.sh]: Added Language and Travis check.

File: r/restic/restic_ubi8.4.sh (hunk 4 above)
Comment (vikasgupta8) [reply r/restic/restic_ubi8.4.sh]: The PAKAGE_PATH is used to perform go operations and cd to the directory.

cd $(ls -d $GOPATH/pkg/mod/$PACKAGE_PATH@$PACKAGE_VERSION)

**Review by seth-priya — APPROVED** [review]

**Pull request merged**